micropub.php

```diff
@@ -469,9 +477,13 @@ private static function update() {
 		// Store metadata from Microformats Properties
 		$args = static::store_mf2( $args );
 		$args = static::store_geodata( $args );
-		// Generate Post Content
-		$args = static::generate_post_content( $args );
 
+		// Generate Post Content
+		$post_content = isset( $args['post_content'] ) ? $args['post_content'] : '';
```

Comment:
unfortunate that this block is duplicated both here and in `create`. maybe not quite enough to warrant unifying them though. meh.

Reply:
I thought of that